code/code/cmd/cmd_attribute.cc

```diff
@@ -496,36 +541,36 @@ const char *statBonusDescr(int percent)
     return "a superb";
   else
     return "an unknown";
-}
-
-
+}  // statBonusDescr
 
-void TBeing::doAttribute(const char *arg)
-{
+void TBeing::doAttribute(const char *arg) {
   sstring buf, buf2, cmdbuf;
   time_info_data playing_time;
   time_info_data birth_data;
   int day;
   Descriptor *d;
 
-  if (!desc)
-    return;
+  if (!desc) return;
 
   one_argument(arg, cmdbuf);
   if (cmdbuf.empty()) {
-    sendTo("Syntax: attribute <personal | condition | statistics | reset>\n\r");
+    sendTo(
+        "Syntax: attribute <personal | condition | statistics | "
+        "reset>\n\r");
     return;
```

Comment:
This is kinda weird looking - wonder what rule causes it?

Reply:
that's ugly. Added `BreakStringLiterals: false`